modules/ROOT/pages/appendix/gql-conformance/additional-cypher.adoc

```diff
@@ -507,29 +507,17 @@ GQL supports `GRAPH TYPES` as a way of constraining a graph schema, but does not
 | Cypher feature
 | Description
 
-| xref:constraints/examples.adoc#constraints-examples-node-uniqueness[Node property uniqueness constraints]
-| Ensures that certain nodes have a set of specified properties whose combined value is unique when all properties exist on the node
+| xref:constraints/managing-constraints.adoc#create-uniqueness-constraints[Uniqueness constraints]
+| Ensure that property values are unique for all nodes with a specific label or all relationships with a specific type. 
```

Comment:
I think "relationships with a specific type" implies that we call them "relationship types" - "relationships with a specific relationship type" is word soup, no?

Reply:
a bit yes, I'm just worried it'll be very close to "relationship of a specific edge type" to mean they match the edge type in the graph type 🤷 (I know they'll not be named edge but I'm gonna use the working name until we have settled on what to call them)

but I'm fine leaving it as type for now and if it becomes a problem later it can be addressed then